www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx

```diff
@@ -621,10 +623,22 @@ type WorkflowInput = {
 const createSubscriptionWorkflow = createWorkflow(
   "create-subscription",
   (input: WorkflowInput) => {
-    const order = completeCartWorkflow.runAsStep({
+    const { id } = completeCartWorkflow.runAsStep({
       input: {
-        id: input.cart_id,
+        id: input.cart_id
+      }
+    })
+
+    const order = useRemoteQueryStep({
```

Comment:
thought: Should we use `useQueryStep` so we slowly migrate to use the `query.graph` everywhere?

Reply:
Oh I wasn't aware of it, we definitely should. Will do it in a separate PR 👍